Sources/web3swift/Web3/Web3+Personal.swift

```diff
@@ -13,12 +13,12 @@ extension Web3.Personal {
      *Locally or remotely sign a message (arbitrary data) with the private key. To avoid potential signing of a transaction the message is first prepended by a special header and then hashed.*
 
      - parameters:
-         - message: Message Data
-         - from: Use a private key that corresponds to this account
-         - password: Password for account if signing locally
+     - message: Message Data
+     - from: Use a private key that corresponds to this account
+     - password: Password for account if signing locally
```

Comment:
This is actually wrong and doesn't produce the correct documentation format. Must be rewritten.

Reply:
Could we disable this rule for a while?

Reply:
Or do you mean the comment as a whole?

Reply:
This is probably due to using `/** */` instead of `///` multi-line comments?